### PR TITLE
contrib/agentsvg: Updated to new CFEngine blue color

### PR DESCRIPTION
**Pull request by olehermanse**
Also made a bunch of smaller improvments and fixes

### CODE REVIEW

File: contrib/agentsvg/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# CFEngine agent SVG / logo generator
+
```
Comment (nickanderson) [inline contrib/agentsvg/README.md:2]: Seems like the README.md should include ` cf-agent, the CFEngine mascot`

Comment (olehermanse) [reply contrib/agentsvg/README.md:2]: Done.

**Review by vpodzime — APPROVED** [review]
Looks good to me. Nice improvements!

**Review by larsewi — APPROVED** [review]
Cool !

**Pull request merged**